pom.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <packaging>jar</packaging>
+  <groupId>io.replikativ</groupId>
+  <artifactId>hitchhiker-tree</artifactId>
+  <version>0.1.10-SNAPSHOT</version>
```

Comment:
Do you want to release a SNAPSHOT first? Then this should be merged into development branch. @whilo